### ISSUE TITLE
Development environment: 

### ISSUE THREAD

**Issue by giandrea77** [openmhealth/shimmer#96]
Hi,

is there a way to import and compile the shimmer project on an Ide like Eclipse or IntelliJ? 

tnx,
Andrea

**Comment by HeenaAgg** [comment]
Yes.... U should have Gradle from market place in your eclipse. Then import gradle project.. browse to your shimmer. It will download runtime jcenter.bintray.com dependencies and gradle.bin.zip-version. You should not be behind proxy for this. After importing -> go to Gradle task -> Application -> bootrun -> And keep your mongodb instance running. It will show up Application started
